src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Cart() {
+  return (
+    <div>Cart</div>
+  )
```

Comment:
since this is a page, why not name it something like `CartPage` ?

Reply:
I think it is self-understood by using the name i.e Cart that it is a cart page.